lib/cequel/schema/table.rb

```diff
@@ -1,5 +1,6 @@
 # -*- encoding : utf-8 -*-
 require 'stringio'
+require 'set'
```

Comment:
On this.. i saw it was used in the class.. in troubleshooting, i wanted to be sure that things were OK.. i will remove in the revision.